docs/configuration/http_conn_man/route_config/route.rst

```diff
@@ -156,6 +157,13 @@ priority
   *(optional, array)* Specifies a set of rate limit configurations that could be applied to the
   route.
 
+.. _config_http_conn_man_route_table_route_exclude_vh:
+
+exclude_vh_rate_limits
```

Comment:
If the default is true, I would invert this to be "include_vh_rate_limits" and make the default false. I think that is more intuitive.